.github/workflows/srtool-build.yml

```diff
@@ -0,0 +1,87 @@
+name: Srtool build
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - 'refs/tags/[0-9]+.[0-9]+.[0-9]+*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
I think the `pull_request` event is not available here. My editor gives me a warning. Should we just use `github.ref` everywhere? For the same PR, it will always be the same, and for different PRs, it will also always be different.

Reply:
Done, used `${{ github.ref_name }}` to make it human readable